launch/proc_image_processing_debug.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+	<node name="proc_image_processing" pkg="proc_image_processing" type="proc_image_processing_node" output="screen" respawn="true" launch-prefix="gdb -ex run --args">
+
+  </node>
+
+</launch>
```

Comment:
Is this file supposed to stay ?

Reply:
yes, it will make the debug way easier in the futur